src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeshipOverlapAlert.cshtml

```diff
@@ -0,0 +1,88 @@
+﻿@model SFA.DAS.ProviderCommitments.Web.Models.DraftApprenticeshipOverlapAlertViewModel
+@{
+    ViewBag.Title = "Training Date Overlap Alert";
+    ViewBag.GaData.Vpv = "/unapproved/add/apprenticeship/overlap-alert";
+}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <partial name="_ValidationSummary" />
+
+        <h1 class="govuk-heading-xl">There may be a problem with the proposed training details</h1>
+
+        <h2 class="govuk-heading-m">Proposed training details</h2>
+        <p>Check you have entered the correct ULN, check the dates you have entered are correct.</p>
+
+        <div>
+            <dl class="govuk-summary-list">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Planned start date
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.StartDate.ToString("Y")
+                    </dd>
+                    <dd class="govuk-summary-list__actions">
+                        @Html.ActionLink("Change", null, null, null, new { href = Context.Request.Headers["Referer"].ToString() })
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Projected finish date
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.EndDate.ToString("Y")
+                    </dd>
+                    <dd class="govuk-summary-list__actions">
+                        @Html.ActionLink("Change", null, null, null, new { href = Context.Request.Headers["Referer"].ToString() })
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Unique learning number
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.Uln
+                    </dd>
+                    <dd class="govuk-summary-list__actions">
+                        @Html.ActionLink("Change", null, null, null, new { href = Context.Request.Headers["Referer"].ToString() })
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Apprentice name
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                        @Model.FirstName @Model.LastName
+                    </dd>
+                    <dd class="govuk-summary-list__actions">
+                        @Html.ActionLink("Change", null, null, null, new { href = Context.Request.Headers["Referer"].ToString() })
```

Comment:
change links should be formatted like this:

https://cs.github.com/SkillsFundingAgency/das-providercommitments/blob/930e40f0c5f80d7fdb7cc24428070bdd68b43399/src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Confirm.cshtml#L120

Specifying the route we want the link to go to rather than using `Context.Request.Referer`

Reply:
updated